query-engine/query-engine/src/exec_loader.rs

```diff
@@ -63,6 +61,11 @@ async fn postgres(
 
     let psql = PostgreSql::from_source(source).await?;
 
+    let force_transactions = params
```

Comment:
Different name here maybe that is not mixed with the old functionality?
(Assuming this also triggers the additional query)

Reply:
Yeah, we still need to pass a boolean flag to the core to force txs. This variable is named to inform why we load the setting from the connection string.